lib/config.js

```diff
@@ -313,6 +313,10 @@ const conf = convict({
       doc: 'Bytes of generated client_secrets',
       default: 32
     },
+    codeVerifier: {
+      doc: 'Bytes of generated PKCE code_verifier',
+      default: 32
```

Comment:
I'm not sure it's useful having this in config, as we don't control the generation of this token.  The spec says we need to support "a minimum length of 43 characters and a maximum length of 128 characters."